lumicks/pylake/group.py

```diff
@@ -12,25 +12,32 @@ class Group:
         The underlying h5py group object
     """
 
-    def __init__(self, h5py_group):
+    def __init__(self, h5py_group, *, lk_file):
```

Comment:
could you add some docstring for `lk_file` please, it took me a while to figure out what was going on

Reply:
Good point! I will add some and the reason it needs to be provided.